docs/.vuepress/components/Button.vue

```diff
@@ -0,0 +1,115 @@
+<template>
+  <!-- <h1>Test</h1> -->
+  <router-link v-if="isRouter" :to="to" :class="buttonClasses">
+    <slot />
+  </router-link>
+  <a href v-else-if="isA" :class="buttonClasses" :target="target" :rel="rel">
```

Comment:
This `href` is broken - I think you want `:href="to"`

Reply:
fixed